### DISCUSSION TITLE
Memory increase in MacOS

### DISCUSSION THREAD

**Discussion by MauroRussi** [freeplane/freeplane#1150] [User testimonials]
Hi community, 

I'd like to share here my experience in a Mac trying to increase the memory of Freeplane.

I downloaded and installed Freeplane on a MacBook Pro Intel (Ventura 13.3.1).
I tried to generate a PDF file and it failed with an OutOfMemoryError.
Checking Freeplane troubleshooting here [](https://docs.freeplane.org/attic/old-mediawiki-content/Troubleshooting.html) it seems the file to edit is `Freeplane.cfg` located into `Applications / Freeplane.app / Contents / app /`. I edit the `Xmx` parameter to `2G` and it works.

It would be good to update the Troubleshooting page for future reference. 

Thanks!

**Comment by wiwo192** [comment]
+1. Please help take a look. This issue pretty much makes it usable on Macbook Air M1 w/ 8GiB of Ram.

Really appreciate it.
